### ISSUE TITLE
Tutorial "02_EBD_preprocessing_and_vanilla_colab": Old imports

### ISSUE THREAD

**Issue by GiuGiglio** [KrishnaswamyLab/MIOFlow#27]
Step #1 Import Libraries: 
Imports 
"from MIOFlow.plots import plot_losses
from MIOFlow.utils import set_seeds "
do not exist anymore and seem to have stayed from a previous version of MIOFlow.

Also, set_seeds(0) does not work.

Fixed it with replacing the code with this:
"import os
import urllib.request
import zipfile
import shutil

import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns

import scanpy as sc
import phate
import torch

# The only MIOFlow module that survived the 2.0 refactor
from MIOFlow.mioflow import MIOFlow

# Replaces the deleted `from MIOFlow.utils import set_seeds`
torch.manual_seed(0)
np.random.seed(0)

use_cuda = torch.cuda.is_available()
print(f'Using CUDA: {use_cuda}')"
